e2e/e2e_test.go

```diff
@@ -368,3 +372,100 @@ func TestPVCResize(t *testing.T) {
 	}
 	steps.Run(t)
 }
+func TestRollingRestart(t *testing.T) {
+
+	// Restart cluster nodes in a rolling cadence, one after the other
+
+	if parallel {
+		t.Parallel()
+	}
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
+
+	testLog := zapr.NewLogger(zaptest.NewLogger(t))
+
+	actor.Log = testLog
+
+	sb := testenv.NewDiffingSandbox(t, env)
+	sb.StartManager(t, controller.InitClusterReconcilerWithLogger(testLog))
+
+	builder := testutil.NewBuilder("crdb").WithNodeCount(3).WithTLS().
+		WithImage("cockroachdb/cockroach:v20.2.5").
```

Comment:
We need an enhancement to the testing framework that pulls this from a central place we can keep up to date.  There are relatively few tests where we should be pinning to specific releases and those are largely around upgrades